src/plugins/intel_cpu/src/nodes/executors/dnnl/dnnl_fullyconnected.hpp

```diff
@@ -69,6 +69,26 @@ class DnnlFCExecutor : public Executor {
         return m_primitive ? m_primitive->implType() : undef;
     }
 
+    void moveMemToNumaNode(int numaNodeID) override {
+        if (curNumaNode == numaNodeID) {
+            return;
+        }
+        const auto newPrimMemDesc = m_primitive->scratchPadDesc();
+        m_scratchPadMemory = m_context->getScratchPad(numaNodeID)->createScratchPadMem(*newPrimMemDesc);
+        m_primArgs[DNNL_ARG_SCRATCHPAD] = m_scratchPadMemory->getPrimitive();
+
+        if (m_primArgs.count(DNNL_ARG_WEIGHTS)) {
+            if (!mbind_move(m_primArgs[DNNL_ARG_WEIGHTS], numaNodeID))
+                std::cout << "move DNNL_ARG_WEIGHTS to node " << numaNodeID << " failed\n";
```

Comment:
Should we throw here?

Reply:
> Should we throw here?

we cannot throw the exception here but I can leave a `DEBUG_LOG` here.